lib/pbench/server/api/resources/pbench_users.py

```diff
@@ -112,8 +117,21 @@ def post(self):
                 }
                 return make_response(jsonify(response_object))
 
+            except (
+                SQLAlchemyError,
+                DataError,
+                DatabaseError,
+                DBAPIError,
+                CompileError,
```

Comment:
Any chance that one of these is a parent/superset of some/all of the others so the list could be shortened?

Reply:
I will take a look into that